<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="main0">
     <widget class="QLineEdit" name="titleEdit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>341</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HY중고딕</family>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="text">
       <string>파충류 사육장</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGraph">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>200</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>실시간 그래프</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExit">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>360</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>나가기</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>261</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/newPrefix/img1.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/newPrefix/img2.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>50</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/newPrefix/img3.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>200</y>
        <width>211</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/newPrefix/img4.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="graph1">
     <widget class="QLabel" name="GraphTitle">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="text">
       <string>실시간 그래프</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="graphWidget" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>601</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEnit">
      <property name="geometry">
       <rect>
        <x>664</x>
        <y>62</y>
        <width>91</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>그래프초기화</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBackGraph">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>440</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>나가기</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../../zxczxczxccxzzcxczxzcx/imgtest.qrc"/>
 </resources>
 <connections/>
</ui>
